airflow/bin/cli.py

```diff
@@ -222,6 +223,52 @@ def pool(args):
         print("Pool {} deleted".format(name))
 
 
+@cli.action_logging
+def pools(args):
+    """
+    set a batch of Airflow pools
+    If a pool is specified in yaml file without slot_counts,
+    it will be default to 3
+
+    :param args: filepath to airflow pools yaml file
```

Comment:
nit: args is a parsed args object as produced by ArgParse. It should contain a filepath property.

Reply:
sure. I will update it more accurately